addons/resource/models/resource.py

```diff
@@ -162,7 +162,7 @@ def default_get(self, fields):
             company_id = res.get('company_id', self.env.company.id)
             company = self.env['res.company'].browse(company_id)
             company_attendance_ids = company.resource_calendar_id.attendance_ids
-            if company_attendance_ids:
+            if not company.resource_calendar_id.two_weeks_calendar and company_attendance_ids:
                 res['attendance_ids'] = [
```

Comment:
@thle-odoo why won't we instead copy missing values as well, like `week_type` or `two_weeks_calendar`? :thinking: